src/VisualStudio/Core/Def/Implementation/TableDataSource/AbstractTableControlEventProcessorProvider.cs

```diff
@@ -42,10 +42,12 @@ public override void PreprocessNavigate(ITableEntryHandle entryHandle, TableEntr
                     return;
                 }
 
-                // we always mark it as handled if entry is ours
-                e.Handled = true;
-
-                roslynSnapshot.TryNavigateTo(index, e.IsPreview);
+                // don't be too strict on navigation on our item. if we can't handle the item,
+                // let default handler to handle it at least.
```

Comment:
interesting.  so the idea is here that VS has a handler for these types of paths?  and by setting 'false' (if we fail) it will take over?

Reply:
yep. they will just open the file. we usually try to open the file in right context.